ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -108,6 +108,12 @@ ${listenerText}
    * @type {string}
    */
   get installBoundaryText() {
+    if (this.features.isEnabled('byow-pki-hcp-cluster-id')) {
+      return `wget -q https://releases.hashicorp.com/boundary-worker/0.11.3+hcp/boundary-worker_0.11.3+hcp_linux_amd64.zip ;\\
```

Comment:
Are we hard coding the version for now? I saw there was some discussion of having a `latest` link or maybe even an APT release?

Reply:
Yes. This is a temporary solution.